### PR TITLE
Add icons — Emacs CN, Cute Doom, Lua, Fortran, GNU Octave, Docker

### LINKED ISSUE #79: Add icons — Emacs CN, Cute Doom, Lua, Fortran, GNU Octave, Docker
Seeing as the icon cap on Discord applications has been increased to 300, here are a few more icons. Also, the Terraform icon has been updated to meet Discord's 512x512 minimum resolution.

Emacs Dragon (see [Emacs China](https://emacs-china.org/))
emacs_dragon_icon
Doom Cute (also referenced in #49)
doom_cute_icon
Lua
lua-mode_icon
Fortran
fortran-mode_icon
GNU Octave
octave-mode_icon
Docker(file)
dockerfile-mode_icon

### PR DESCRIPTION
**Pull request by Feyorsh**
Fixes #79

### CODE REVIEW

File: elcord.el [hunk 1]
```diff
@@ -79,12 +79,14 @@ See <https://discordapp.com/developers/applications/me>."
...
                                     (comint-mode . "comint-mode_icon")
                                     (cperl-mode . "cperl-mode_icon")
+                                    (dockerfile-mode . "dockerfile-mode_icon")
                                     (elixir-mode . "elixir-mode_icon")
                                     (emacs-lisp-mode . (elcord--editor-icon))
                                     (enh-ruby-mode . "ruby-mode_icon")
                                     (erc-mode . "irc-mode_icon")
                                     (erlang-mode . "erlang-mode_icon")
                                     (forth-mode . "forth-mode_icon")
+									(fortran-mode . "fortran-mode_icon")
```
Comment (Mstrodl) [inline elcord.el:89]: Fix indentation pls

**Review by Mstrodl — CHANGES_REQUESTED** [review]

**Comment by skykanin** [comment]
@Feyorsh if you like I can apply the necessary changes in a new PR to get this merged if you don't have the time

File: elcord.el [hunk 2]
```diff
@@ -98,12 +100,14 @@ See <https://discordapp.com/developers/applications/me>."
                                     (go-mode . "go-mode_icon")
                                     (latex-mode . "latex-mode_icon")
                                     (lisp-mode . "lisp-mode_icon")
+									(lua-mode . "lua-mode_icon")
```
Comment (skykanin) [inline elcord.el:103]: You forgot to fix the indentation here

File: elcord.el [hunk 3]
```diff
@@ -98,12 +100,14 @@ See <https://discordapp.com/developers/applications/me>."
...
                                     (magit-mode . "magit-mode_icon")
                                     (markdown-mode . "markdown-mode_icon")
                                     (meson-mode . "meson-mode_icon")
                                     (nim-mode . "nim-mode_icon")
                                     (nix-mode . "nix-mode_icon")
                                     (ocaml-mode . "ocaml-mode_icon")
+									(octave-mode . "octave-mode_icon")
```
Comment (skykanin) [inline elcord.el:110]: and here

**Review by skykanin — CHANGES_REQUESTED** [review]

**Comment by Feyorsh** [comment]
Untabify user error 🙃 
Thanks for pointing that out @skykanin, should be good to go now

**Review by skykanin — APPROVED** [review]

**Pull request merged**